modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -1075,6 +1076,9 @@ public class IgniteImpl implements Ignite {
                 clockService
         );
 
+        EventLog eventLog = new EventLogImpl(clusterCfgMgr.configurationRegistry()
```

Comment:
we should have just a single EventLog, so let's a little bit refactor the existing code here and use the same instance here and for AuthenticationManager

Reply:
Thanks. I hope I fixed it.